### DISCUSSION TITLE
Reading  VTK files - directing the DataLoader to use the ITKReader

### DISCUSSION THREAD

**Discussion by andreabnes** [Project-MONAI/tutorials#1183] [Q&A]
Hi,

I am new to MONAI. I am modifying the UNETR BraTS21 tutorial to work with different data. I am looking to feed some CT volumes saved as VTK files.

I have modified the example which now builds training and validation datasets pointing to my own files. The DataLoader fails with an exception indicating that pydicom is being used to read the files, and pydicom fails as those files are not DICOM. The ITKReader, which is part of MONAI, would be able to handle VTK files. I am unsure about how to tell the DataLoader to use the ITKReader instead of pydicom. The extension of the files is *.VTK, but that does not seem to have an influence.

Does anyone have a tip?

Thank and Best Regards,

Andrea

**Comment by andreabnes** [comment]
I am experimenting with passing the ITKReader as a parameter to a LoadImaged transform. I will report results.

Andrea

**Reply by ericspod** [reply]
Yes, Passing the reader as a parameter will force it to be used. If this uses VTK and your images don't load properly I would suggest converting them to nifti if that's possible and not difficult.
